include/fuse_common.h

```diff
@@ -82,8 +82,12 @@ struct fuse_file_info {
 	    nothing when set by open()). */
 	unsigned int cache_readdir : 1;
 
+	/** Can be filled in by open, to indicate that the file is
+	    stream-like (no file position at all). */
```

Comment:
Could you state if this must be specified together with `nonseekable`?

Also, is there a way to *briefly* explain when one should enable this and when one should go with just `nonseekable`? (Maybe with one example of a file of each kind)

Reply:
thanks, this makes sens. I suggest to go "both must be set way" + see interdiff proposed in main reply.